From cb6511354b500d1b1bd8ff140fdf0ea106b174d4 Mon Sep 17 00:00:00 2001
From: David Tardon <dtardon@redhat.com>
Date: Wed, 3 Sep 2014 16:20:40 +0200
Subject: [PATCH] drop useless test for ant-apache-regexp

It has not been needed since commit
1de48c417404464ca1e34e5c5d1c82a9342349bb 4 years ago.

Change-Id: I53ceb5d8d6c02c7a13c86cdd884e4fc378a2c492
---
 configure.ac | 44 --------------------------------------------
 1 file changed, 44 deletions(-)

diff --git a/configure.ac b/configure.ac
index f65ab66..897cb1e 100644
--- a/configure.ac
+++ b/configure.ac
@@ -12311,50 +12311,6 @@ EOF
         AC_MSG_ERROR([no, you need at least Ant >= $ant_minver])
     fi
 
-    if test "$ENABLE_MEDIAWIKI" = "TRUE"; then
-        AC_MSG_CHECKING([whether Ant supports mapper type="regexp"])
-        rm -rf confdir
-        mkdir confdir
-        cat > conftest.java << EOF
-            public class conftest {
-                int testmethod(int a, int b) {
-                    return a + b;
-                }
-            }
-EOF
-
-        cat > conftest.xml << EOF
-            <project name="conftest" default="conftest">
-            <target name="conftest" depends="copytest">
-                <javac srcdir="." includes="conftest.java">
-                </javac>
-            </target>
-            <target name="copytest">
-                 <copy todir="confdir">
-                 <fileset dir="confdir" includes="**/*.abc" casesensitive="yes"/>
-                 <filterset/>
-                 <mapper type="regexp" from="^(.*[/\\])foo([/\\].*)" to="\1baa\2"/>
-                 </copy>
-            </target>
-            </project>
-EOF
-
-        if test "$JAVACISGCJ" = "yes"; then
-            JAVA_HOME=; export JAVA_HOME
-            ant_gcj="-Dbuild.compiler=gcj"
-        fi
-        AC_TRY_COMMAND("$ANT" $ant_gcj -buildfile conftest.xml 1>&2)
-        if test $? = 0 -a -f ./conftest.class; then
-            AC_MSG_RESULT([yes])
-            rm -rf confdir
-        else
-            echo "configure: Ant test failed" >&5
-            cat conftest.java >&5
-            cat conftest.xml >&5
-            rm -rf confdir
-            AC_MSG_ERROR([no. Did you install ant-apache-regexp?])
-        fi
-    fi
     rm -f conftest* core core.* *.core
 fi
 
-- 
1.9.3

